DEV_GUIDE.md

```diff
@@ -340,10 +341,12 @@ has been applied ineffectively.
 * `SKIP_TEARDOWN`: variable for development purposes to avoid keep deploying and deleting deployments each run. Default value: `false`
 * `CONTAINER_CONFIG_PATH`: directory where `config.json` file is located. This file contains the pull secrets to be used by
 the container engine. Default value: `$HOME/.docker/config.json`
-* `VAULT_CHART_VERSION`: version of Vault Helm Chart to be used by the System Tests for Envelope Encryption. Default value: `0.27.0`
+* `VAULT_CHART_VERSION`: version of Vault Helm Chart to be used by the System Tests for Record Encryption. Default value: `0.27.0`
 * `SKIP_STRIMZI_INSTALL`: skip strimzi installation. Default value: `false`
 * `KAFKA_CLIENT`: client used to produce/consume messages. Default value: `strimzi_test_client`. Currently supported values: `strimzi_test_client`, `kaf`, `kcat`
-
+* `AWS_ACCESS_KEY_ID`: key id of the aws account to be used for KMS. Default value: `test`
```

Comment:
So if you leave these two vars unset, system tests will use localstack, right?  This isn't super clear from what you've written here.